src/components/review/ReviewsList.js

```diff
@@ -67,10 +68,20 @@ const AlbumReviews = ({ albumId }) => {
       {user && userHasReviewed && (
         <p> You already submitted the review for this album</p>
       )}
+
       {/* Conditionally render the WriteReview component */}
       {user && !userHasReviewed && (
-        <WriteReview albumId={albumId} refreshReviews={refreshReviews} />
+        <WriteReview
+          albumId={albumId}
+          refreshReviews={refreshReviews}
+          userHasReviewed={userHasReviewed}
+          reviewId={reviews[0]?._id}
+        />
       )}
+       {user && userHasReviewed && (
+          <DeleteReview reviewId={
+            reviews.find((review) => review.user === user?.user?._id)?._id} refreshReviews={refreshReviews} />
```

Comment:
I've updated `useAuth` in `main` branch to no longer need this nested-access (😮‍💨). So now we can simplify accessing the user to:

```suggestion
            reviews.find((review) => review.user === user?._id)?._id} refreshReviews={refreshReviews} />
```